.circleci/config.yml

```diff
@@ -95,6 +95,7 @@ jobs:
       - hugo/install:
           version: "0.78.0"
       - hugo/hugo-build
```

Comment:
You'll need to add a colon here

Reply:
@dsayling Oops, thanks for catching that. Fixed in b2dea790e58091cd25c8a1165737d269b3e9c99e.